src/media/icons/dvc.svg

```diff
@@ -1,8 +1 @@
-</g>
-</svg>
+<svg width="256" height="257" viewBox="0 0 256 256" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M167.189 167.877c-11.856-11.734-17.347-25.954-17.347-42.726 0-16.867 5.46-31.151 17.347-42.98a59.627 59.627 0 0 1 19.857-13.37 58.586 58.586 0 0 1 23.387-4.39c15.093-.253 29.724 5.326 40.965 15.623 1.716 1.595 1.748 3.189.188 4.72l-13.198 13.614c-1.466 1.34-2.902 1.34-4.243 0-6.316-5.663-14.469-8.72-22.87-8.577-9.485 0-17.378 3.38-23.712 10.203a33.843 33.843 0 0 0-6.924 11.266 34.389 34.389 0 0 0-2.124 13.126c0 9.438 2.808 17.473 9.173 24.137a31.864 31.864 0 0 0 10.86 7.546 31.29 31.29 0 0 0 12.914 2.434c8.265.19 16.319-2.674 22.683-8.067 1.56-1.339 3.026-1.275 4.368.159l13.229 13.966c1.466 1.467 1.404 2.965-.156 4.528-11.072 10.775-25.868 16.633-41.153 16.293a58.985 58.985 0 0 1-23.349-4.321 60.016 60.016 0 0 1-19.895-13.216v.032Z" fill="url(#a)"/><path d="M3.202 181.875V67.727a3.12 3.12 0 0 1 .223-1.267c.161-.401.404-.763.713-1.06.26-.305.582-.547.943-.712.36-.166.752-.249 1.147-.245h40.779C63.73 65.718 76.21 70.5 87.692 82.298c11.544 11.798 16.13 25.189 17.596 42.407 1.404 16.58-5.366 30.928-16.848 42.726a56.309 56.309 0 0 1-18.904 13.303 55.248 55.248 0 0 1-22.53 4.488H6.229a2.777 2.777 0 0 1-1.144-.253 2.847 2.847 0 0 1-.946-.703 3.124 3.124 0 0 1-.714-1.078 3.184 3.184 0 0 1-.222-1.282v-.031Zm26.083-86.408v64.184h16.037c9.36 0 17.16-3.38 23.338-10.108 6.177-6.716 9.508-15.655 9.266-24.87 0-9.788-3.12-18.047-9.266-24.774a30.683 30.683 0 0 0-10.6-7.633 30.095 30.095 0 0 0-12.738-2.475H29.285v5.676Z" fill="url(#b)"/><path d="m153.118 104.362-25.272 56.755h-1.03l-26.988-60.708c3.058 7.269 4.68 15.241 5.46 24.264 1.03 12.116-2.308 23.021-8.642 32.65l26.832 58.764c.686 1.275 1.622 1.913 2.87 1.913h1.685c1.248 0 2.184-.606 2.839-1.881l27.332-59.465c-5.688-9.475-8.591-20.421-8.362-31.535 0-7.397 1.061-14.348 3.276-20.757Z" fill="url(#c)"/><defs><linearGradient id="a" x1="201.321" y1="124.981" x2="201.321" y2="185.396" gradientUnits="userSpaceOnUse"><stop stop-color="#F46837"/><stop offset="1" stop-color="#DD5236"/></linearGradient><linearGradient id="b" x1="54.336" y1="124.981" x2="54.336" y2="185.227" gradientUnits="userSpaceOnUse"><stop stop-color="#13ADC7"/><stop offset="1" stop-color="#1497B8"/></linearGradient><linearGradient id="c" x1="104.6" y1="160.055" x2="104.6" y2="217.447" gradientUnits="userSpaceOnUse"><stop stop-color="#7E51C2"/><stop offset="1" stop-color="#945DD6"/></linearGradient></defs></svg>
```

Comment:
`width="256" height="257" viewBox="0 0 256 256"` doesn't seem consistent

Reply:
Height should be `256`. Maybe it was some kind of bug when we created the svgs. I'll update the pull request and I'll mention it the slack thread I made about updating `static`.